modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -100,10 +100,16 @@ def generate_pdf # rubocop:disable Metrics/MethodLength
 
         def synchronous
           auto_claim = shared_submit_methods
-          # This kicks off the first of three jobs required to fully establish the claim
-          process_claim(auto_claim, false) unless Flipper.enabled? :claims_load_testing
 
-          auto_claim.reload
+          unless claims_load_testing || sandbox_request(request) || mocking
+            pdf_generation_service.generate(auto_claim&.id, veteran_middle_initial)
+            docker_container_service.upload(auto_claim&.id)
+            queue_flash_updater(auto_claim.flashes, auto_claim&.id)
+            start_bd_uploader_job(auto_claim) if auto_claim.status != errored_state_value
+            auto_claim.reload
+          end
```

Comment:
this needs a bit of rework. 
1. all of the calls need to be voided for `claims_load_testing` is enabled so we don't build a PDF, establish a claim, or upload said PDF so we don't max out all our test users.
2. the pdf-generation is the only one we need to ignore for `mocking` as it's the only one we really care to. The rest are already set up to be mocked, so they'll respond appropriately. - this would eliminate the need for line :112 as well.
3. `sandbox_request` is kind of a scary check for me. we'll have to redeploy the entire application to make a change there where flipper can be instantly toggled and the settings update every X minutes, so we could modify either the Settings or Flipper near real time.

Reply:
@FonzMP the sandbox_request just looks at the request.base_url, I do not know where the Flipper comes into play:

        def sandbox_request(request)
          request.base_url == 'https://sandbox-api.va.gov'
        end
        
To verify, what are the steps we do not want to take if the request comes from Sandbox?